### PR TITLE
Add Get-WebListenerUrl Based Examples to WebListener README.md

### LINKED ISSUE #4918: Add Get-WebListenerUrl examples to WebListener App documentation
per https://github.com/PowerShell/PowerShell/pull/4905#pullrequestreview-65030961

Need to add `Get-WebListenerUrl` examples to the WebListener App `REAMDME.md`

### PR DESCRIPTION
**Pull request by markekraus**
closes #4918

* Switches from URL Examples to Get-WebListenerUrl
* Minor formatting
* Correct the Compression examples

@dantraMSFT As requested in #4905

### CODE REVIEW

File: test/tools/WebListener/README.md [hunk 1]
```diff
@@ -38,6 +38,11 @@ Returns a static HTML page containing links and descriptions of the available te
 
 Returns a JSON object containing the details of the Client Certificate if one is provided in the request.
 
+```powershell
+$uri = Get-WebListenerUrl -Test 'Compression' -Https -Certificate $certificate
```
Comment (TravisEz13) [inline test/tools/WebListener/README.md]: Is the test supposed to be compression?

Comment (markekraus) [reply test/tools/WebListener/README.md]: Ooops.. fixed.

**Comment by TravisEz13** [comment]
restarted macOS CI.  if it doesn't fix the issue, you might need to rebase.

**Comment by daxian-dbw** [comment]
@TravisEz13 Can you please update your review?

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**